### DISCUSSION TITLE
parameter values not coming in emails integrated via email  documents

### DISCUSSION THREAD

**Discussion by shubhangi92** [pimcore/pimcore#10392] [General]
Hi ,

Not getting values of the parameter we pass in email document like %Text(name); and its coming same in emails also instead of value of it.

This I found in recent pimcore version not in pimcore earlier version.

Thanks
Shubhangi

**Comment by dvesh3** [comment]
Pimcore X does not support placeholders like `%Text(name);`. please use twig parameter `{{ name }}` in the email template. thanks!
